app/views/group_offers/_form.html.slim

```diff
@@ -26,16 +26,6 @@
 
   .row
     .col-xs-12.col-md-4
-      = f.input :necessary_volunteers, collection: 1..10
-    .col-xs-12.col-md-8
-      - if policy(GroupOffer).supervisor_privileges?
-        = f.association :volunteers
-      - else
-        = f.association :volunteers,
-          collection: Volunteer.without_active_assignment.not_in_any_group_offer.not_responsible
-
-  .row
-    .col-xs-12.col-md-4
       - if policy(GroupOffer).supervisor_privileges?
```

Comment:
Although that was not introduced in this PR, has to be removed. Department managers can also add volunteer responsible.

Reply:
I think the opposite. Thats what a policy is for. 
This  policy:

```ruby
alias_method :supervisor_privileges?, :superadmin?
```

can be switched to:

```ruby
alias_method :supervisor_privileges?, :superadmin_or_department_manager?
```

So the policy can be adapted, and the views can stay the same.